style/icons/shield40_us_interstate_2.svg

```diff
@@ -5,38 +5,60 @@
    version="1.0"
```

Comment:
Please save the SVGs as "optimized SVG" in Inkscape to save on space.

Reply:
This is addressed in efa8f21f35f3d01a68bcf3d04cbda7553f26e7d5